### DISCUSSION TITLE
Consider using bluetuith for bluetooth

### DISCUSSION THREAD

**Discussion by taschenlampe** [openSUSE/openSUSEway#87] [Ideas]
"[Bluetuith](https://github.com/darkhz/bluetuith) is a TUI-based bluetooth connection manager, which can interact with bluetooth adapters and devices. It aims to be a replacement to most bluetooth managers, like blueman."

Bluetuith is  labeled as alpha, still I found it to be stable and quite useful. I think that this tool could be the default cli-tool for bluetooth management in openSuseWay.  What do you think about this tool?

**Comment by denisok** [comment]
tool looks good, will try it out.

Adding it to the pattern s not a problem, the question is how to integrate and connect it with waybar probably.

Also there is no such package yet in Factory.

**Comment by jubalh** [comment]
> Also there is no such package yet in Factory.

Done: https://build.opensuse.org/package/show/home:jubalh/bluetuith (untested)
Does anybody have an idea for a suitable devel project?

And would be great if somebody else could (help) maintain it.

**Comment by jubalh** [comment]
bluetuith devel project: https://build.opensuse.org/package/show/hardware/bluetuith
SR to Factory: https://build.opensuse.org/request/show/1058661

If you want to help maintaining please send a request. That would be appreciated :)

**Comment by denisok** [comment]
BTW - I like new Bluetooth functionality in openSUSEway. I tried to click on icon and that gets me to the `bluetoothctl` which I found is hard to use. I tried `bluetoothctl` and it worked much better for me.

**Reply by FilippoBonazziSUSE** [reply]
Did you mean `bluetuith` worked much better? Let me try it as well

**Reply by denisok** [reply]
yes, worked much better for me. At least I was able to connect to my headset. When with `ctl` I spent couple of minutes and got lost.

**Comment by FilippoBonazziSUSE** [comment]
I've started using bluetuith and I do find it very easy and effective. Do we want to add to our `Recommends` or `Suggests`? not sure about `Requires`

**Comment by FilippoBonazziSUSE** [comment]
This was done in #207
